hazelcast/src/test/java/com/hazelcast/client/test/CustomComparator.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2008, Hazelcast, Inc. All Rights Reserved.
```

Comment:
The copyright should include the year range, so `Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.`